models/gan_networks.py

```diff
@@ -266,14 +273,22 @@ def define_D(
     f_s_semantic_nclasses,
     model_depth_network,
     train_feat_wavelet,
+    G_unet_mha_num_head_channels,
+    G_unet_mha_res_blocks,
+    G_unet_mha_channel_mults,
+    G_unet_mha_norm_layer,
+    G_unet_mha_group_norm_size,
     **unused_options
 ):
 
     """Create a discriminator
 
     Parameters:
         model_input_nc (int)     -- the number of channels in input images
+        model_output_nc (int)     -- the number of channels in output images
         D_ndf (int)          -- the number of filters in the first conv layer
+        D_ngf(int)  -- the number of filters in the last cov layer
```

Comment:
Don't change this, also I believe D_ngf doesn't exist.

Reply:
D_ngf is one parameter in the first unet (    https://github.com/jolibrain/joliGEN/blob/98a09fe6360b396cd3d309227deef47d278cf3c3/models/modules/unet_architecture/unet_generator.py#L14C16-L14C16      )

Reply:
it is being removed